### ISSUE TITLE
Resonance Archive armor stands give crystalized minerals and cloth

### ISSUE THREAD

**Issue by MtSheepFarm** [anegostudios/VintageStory-Issues#4817]
### Game Version

1.20.0 rc6

### Platform

Windows

### Modded

Modded

### SP/MP

Singleplayer

### Description

I've just made it into the archives for the first time in 1.20, and the aged dummies in the guard room appear naked, but give a couple misc. crystalized minerals and bolts of cloth with empty durability bars beneath them. I've included a screenshot displaying this as well. Please let me know if I can give any more information which might be helpful. 

Thanks!

### Screenshots

![image](https://github.com/user-attachments/assets/f198c3a4-29fa-4db4-b579-f7a3b8dedc0f)

**Comment by radfast** [comment]
This may be a duplicate of #4776, not sure yet.

@MtSheepFarm, did you start this world in 1.20.0-rc.5 or any earlier version?  Is there any possibility that the game already generated parts of the Resonance Archives before you updated the game to -rc.6?

In the affected room, please try running command `/wgen regen 0`.  (You may want to be in Creative Mode, `/gm 2`, before trying this, just in case of falling)  It should briefly destroy and then regenerate the vertical column of chunks you are standing in, including that room.  If all goes well, that should fix the issue.

If that does not fix the issue then we have a more serious problem on our hands....

**Comment by MtSheepFarm** [comment]
That command did fix the issue, thank you! I had generated the world in rc1, so that must have been the problem.

**Comment by radfast** [comment]
> That command did fix the issue, thank you! I had generated the world in rc1, so that must have been the problem.

Awesome, thank you for confirming.  Then I believe this did have the same cause as #4776, and it's one of the issues which we fixed in 1.20.0-rc.6.  Great to know the fix is working properly.

**Comment by EvanBalster** [comment]
I just encountered this issue in 1.20.3 stable.  Right-clicking on the aged armor stands in the guardroom, I received:

* NPC Guide Stick
* Rotten Lore Book (blue)
* Rotten Lore Book (brown)
* Envelope sealed with blue wax

All four items showed a healthbar seemingly representing a negative number.  The last three items could not be 'read' by right-clicking.  I kept the rotten books in a bookshelf for decoration.
